cardano-api/src/Cardano/API.hs

```diff
@@ -200,6 +201,7 @@ module Cardano.API (
 
     -- * Errors
     Error,
+    displayError,
```

Comment:
Oh oops, yes this should have already been exported.

We should probably just export `Error (..)` which would include `displayError` right? It's ok if other modules were to add their own instances. We don't need to prevent that.